LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2022 OpenMined
```

Comment:
It turned out no need to change this. It's just an explanation how to apply license. Could you please revert changes in this file?